packages/tools/src/Tool.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * A unique string identifier for each type of supported monorepo tool.
+ */
+export type ToolType = "bolt" | "lerna" | "pnpm" | "root" | "rush" | "yarn";
```

Comment:
An additional question related your doubts regarding `.type` - if we don't want people to rely on this etc, should we even export this union?

Reply:
I think not -- I will remove altogether.